docs/datascience/data-wrangler-full-documentation.md

```diff
@@ -0,0 +1,199 @@
+> | pandas | 0.25.2                   | Yes                     |
+
+If they are not found on your environment, Data Wrangler will attempt to install them for you via pip. If Data Wrangler is unable to install dependencies, the easiest workaround is to manually run pip install, and then launch Data Wrangler again. These dependencies are required for Data Wrangler such that it can generate Python and Pandas code.
+
+
+## Opening Data Wrangler
+Anytime you are in Data Wrangler, you are in a **sandboxed** environment, meaning you are safely able to explore and transform the data, without worry of modifying the original dataset until you explicity export your changes.
+
+### Launching Data Wrangler from a Jupyter Notebook
+
+There are 3 ways to launch Data Wrangler from your Jupyter Notebook
+
+![image](https://github.com/microsoft/vscode-docs/assets/15910920/c03a4b85-80cf-46e0-b365-38820d51ff96)
+
+1. In the Jupyter Notebook Variable Explorer, beside any supported data object, you will see a button to launch Data Wrangler.
+2. If you have a Pandas data frame in your Notebook, you’ll now see an `Open 'df'` button (where 'df' is the variable name of your data frame) appear in bottom of the cell after running code that outputs the data frame. This includes 1) `df.head()`, 2) `df.tail()`, 3) `df`.
+3. In the Notebook toolbar, selecting `View data` will bring up a list of every supported data object in your Notebook. You can then choose which variable in that list you want to open in Data Wrangler.
+
+### Launching Data Wrangler directly from a file
+
+You can also launch Data Wrangler directly from a local file (such as a .csv). To do so, open any folder in VS Code that contains the file you’d like to open. In the File Explorer panel, right click the file and click `Open in Data Wrangler`.
+
+![image](https://github.com/microsoft/vscode-docs/assets/15910920/517e1e29-ba45-4e24-87fb-adb53a6207f1)
+
+Data Wrangler currently supports the following filetypes
+1. `.csv`/`.tsv`
+2. `.xls`/`.xlsx`
+3. `.parquet`
+
+Depending on the file type, you will be able to specify the delimeter and/or sheet of the file.
+![image](https://github.com/microsoft/vscode-docs/assets/15910920/424a88fd-be5a-4d03-b878-426fe380266a)
+
+You can also set these file types to open with Data Wrangler by default.
+
+
+## UI Tour
+
+Data Wrangler has 2 modes when working with your data. The details for each mode are explained in the subequent sections below.
+
+1. **View Mode:** The view mode optimizes the interface for you to quickly view, filter and sort your data. This mode is great for doing initial exploration on the dataset.
+2. **Edit Mode:** The edit mode optimizes the interface for you to apply transformations, cleaning, or modifications to your dataset. As you apply these transformations in the interface, Data Wrangler will autoamtically generate the relevant Pandas code, and this can be exported back into your Notebook to be reused.
```

Comment:
```suggestion
2. **Edit Mode:** The edit mode optimizes the interface for you to apply transformations, cleaning, or modifications to your dataset. As you apply these transformations in the interface, Data Wrangler automatically generates the relevant Pandas code, and this can be exported back into your Notebook for reuse.
```